website/docs/components/code-block/partials/code/component-api.md

```diff
@@ -0,0 +1,41 @@
+## Component API
+
+This component uses [prism.js](https://prismjs.com/) under the hood.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="value" @type="string">
+    The text/code content for the CodeBlock.
+  </C.Property>
+  <C.Property @name="language" @type="string" @values={{array "bash" "go" "hcl" "json" "log" "shell-session" "yaml"}}>
+    The coding language to use for syntax highlighting.
```

Comment:
No mention of what to do if a consumer needs a language which is not in this list